examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plotter = paseos.plot(sim, paseos.PlotType.SpacePlot)
</pre><p>why the plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
I forgot to remove it. I will edit it out.